crates/aptos-rosetta/aptos.ros

```diff
@@ -94,7 +94,7 @@ transfer(20){
         print_message({"Find someone to send money with balance minimum": "20000"});
```

Comment:
Update this message? Might be worth using a constant for this, but idk if `.ros` supports that.

Reply:
Yeah, there aren't really "constants".  The constructor DSL isn't too complicated.